README.md

```diff
@@ -145,10 +203,11 @@ The .Net client comes with implementations for the built-in activation strategie
 - ApplicationHostnameStrategy
 - FlexibleRolloutStrategy
 
-Read more about the strategies in [activation-strategy.md](https://docs.getunleash.io/user_guide/activation_strategy).
+Read more about the strategies in [the activation strategy reference docs](https://docs.getunleash.io/reference/activation-strategies).
 
-#### Custom strategies
-You may also specify and implement your own strategies. The specification must be registered in the Unleash UI and you must register the strategy implementation when you wire up unleash. 
+### Custom strategies
+
+You may also specify and implement your own strategies. The specification must be registered in the Unleash UI and you must register the strategy implementation when you wire up unleash.
```

Comment:
Does this deserve a link to https://docs.getunleash.io/reference/custom-activation-strategies?

Reply:
Sure does! Addressed in 75bf7fe